cdb2api/cdb2api.c

```diff
@@ -4467,6 +4467,17 @@ static int cdb2_dbinfo_query(cdb2_hndl_tp *hndl, char *type, char *dbname,
     return -1;
 }
 
+static inline void only_read_config()
+{
+    FILE * fp = fopen(CDB2DBCONFIG_NOBBENV, "r");
+    if (fp != NULL) {
+        read_comdb2db_cfg(NULL, fp, NULL, NULL, NULL,
```

Comment:
this isn't falling back to /bb/bin or other location of comdb2db.cfg..

Reply:
you are right, i was not thinking about  those other locations.